### PR TITLE
Add method for adding not referenced actors

### PR DESCRIPTION
**Pull request by azakhi**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -381,11 +472,22 @@ int main(int argc, char * argv[]) {
 
 	SkyblivionConverter converter = SkyblivionConverter(oblivionCollection, skyrimCollection, std::string(argv[3]));
 
-	std::cout << "Converting QUST records.. " << std::endl;
-	std::vector<Sk::QUSTRecord *> *resQUST = converter.convertQUSTFromOblivion();
 	std::cout << "Converting DIAL records.. " << std::endl;
 	std::vector<Sk::DIALRecord *> *resDIAL = converter.convertDIALFromOblivion();
 
+	/**
```
Comment (Ormin) [inline main.cpp:478]: Why changed order? :)

File: main.cpp [hunk 2]
```diff
@@ -348,6 +348,97 @@ void convertWEAP(SkyblivionConverter &converter) {
 
 }
 
+void addSpeakAsNpcs(SkyblivionConverter &converter, Collection &skyrimCollection) {
+	std::string metadataFile = converter.ROOT_BUILD_PATH() + "Metadata";
```
Comment (Ormin) [inline main.cpp:352]: We'll have to extract the metadata parsing part into more generic class , but for now i suppose its okay

File: main.cpp (hunk 1 above)
Comment (azakhi) [reply main.cpp:478]: If some of the topics are added after quests, when converting quests they do not exist in EDID map. So they are not being bound. While this is not related to adding not referenced actors, to solve the issue with ArenaSpectator quest the topic related to it needs to be bound.

Comment (Ormin) [reply main.cpp:478]: Makes sense - blank QUST records are here in .esm, so converting DIALs first will bind them properly.
Thanks

**Review by Ormin — APPROVED** [review]

**Pull request merged**